src/index.html

```diff
@@ -13,6 +13,47 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars--0">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+    <div class="stars--1">
+      <div class="stars--1__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+    <div class="stars--2">
+      <div class="stars--2__star"></div>
+      <div class="stars--2__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+    <div class="stars--3">
+      <div class="stars--3__star"></div>
+      <div class="stars--3__star"></div>
+      <div class="stars--3__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+    <div class="stars--4">
+      <div class="stars--4__star"></div>
+      <div class="stars--4__star"></div>
+      <div class="stars--4__star"></div>
+      <div class="stars--4__star"></div>
+      <div class="stars__star"></div>
+    </div>
+    <div class="stars--5">
+      <div class="stars--5__star"></div>
+      <div class="stars--5__star"></div>
+      <div class="stars--5__star"></div>
+      <div class="stars--5__star"></div>
+      <div class="stars--5__star"></div>
+    </div>
```

Comment:
Add empty lines between multiline sibling blocks of HTML to add some 'air' and simplify reading. But don't add them between parent and child elements.